src/js/claims-status/containers/DocumentRequestPage.jsx

```diff
@@ -34,18 +34,17 @@ const Element = Scroll.Element;
 class DocumentRequestPage extends React.Component {
   componentDidMount() {
     this.props.resetUploads();
-    if (this.props.trackedItem) {
```

Comment:
Given the new redirect logic, is this check no longer required?

Reply:
I think so. If this component mounts while loading, we wouldn't have a tracked item yet.